conformsTo.js

```diff
@@ -23,8 +23,6 @@ import keys from './keys.js'
  * conformsTo(object, { 'b': function(n) { return n > 2 } })
  * // => false
  */
-function conformsTo(object, source) {
-  return source == null || baseConformsTo(object, source, keys(source))
-}
+const conformsTo = (object, source) =>  source == null || baseConformsTo(object, source, keys(source))
 
```

Comment:
This can remain a declaration ☝️

Reply:
It's just personal preference for the primary function of the module to be a declaration instead of a function expression.

Reply:
Done

Reply:
Thanks, could you do it for the others I didn't explicitly call out too.

Reply:
I did it for what are all the files you requested below.

Reply:
I understood your request properly,I did it properly can you please `pull` the code and check it